source/docs/getting-started/getting-started-frc-control-system/imaging-your-roborio.rst

```diff
@@ -30,6 +30,8 @@ Launching the Imaging Tool
 
 The roboRIO imaging tool and latest image are installed with the NI Update Suite. Launch the imaging tool by double clicking on the shortcut on the Desktop. If you have difficulties imaging your roboRIO, you may need to try right-clicking on the icon and selecting Run as Administrator instead.
 
+.. note:: The roboRIO imaging tool is also located at ``C:\Program Files (x86)\National Instruments\LabVIEW 2019\project\roboRIO Tool``
```

Comment:
```suggestion
.. note:: The roboRIO imaging tool is also located at ``C:\Program Files (x86)\National Instruments\LabVIEW YEAR\project\roboRIO Tool``
```

Reply:
Should be YEAR-1, unfortunately.

Reply:
Right... is there a way to make this year agnostic or do we need to bite the bullet?